### PR TITLE
Document arrow related constructor args

### PR DESCRIPTION
**Pull request by FlorentinD**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library

### CODE REVIEW

File: doc/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
...
+
+[source,python,role=no-test]
+----
+gds = GraphDataScience(
+  NEO4J_URI, 
+  auth=(NEO4J_USER, NEO4J_PASSWORD), 
+  arrow=True, 
+  arrow_disable_server_verification=False, 
+  arrow_tls_root_certs=CERT
+  )
```
Comment (adamnsch) [inline doc/modules/ROOT/pages/getting-started.adoc:112]:
```suggestion
)
```

File: doc/modules/ROOT/pages/getting-started.adoc [hunk 2]
```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
 gds.set_database("my-db")
 ----
 
+=== Configure Arrow parameters
+
+If Arrow is available on the server, we can provide the `GraphDataScience` constructor with several keyword parameters to configure the connection:
+
+* arrow_disable_server_verification - A flag that indicates that, if the flight client is connecting with
```
Comment (adamnsch) [inline doc/modules/ROOT/pages/getting-started.adoc:98]:
```suggestion
* `arrow_disable_server_verification`: A flag that indicates that, if the flight client is connecting with
```
(same suggestion for next arg)

File: doc/modules/ROOT/pages/getting-started.adoc [hunk 3]
```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
 gds.set_database("my-db")
 ----
 
+=== Configure Arrow parameters
+
+If Arrow is available on the server, we can provide the `GraphDataScience` constructor with several keyword parameters to configure the connection:
```
Comment (adamnsch) [inline doc/modules/ROOT/pages/getting-started.adoc:96]: Do we want to add a link/reference to the Arrow docs on the server here? If people don't know what this is about

**Review by adamnsch — APPROVED** [review]
Nice work! Just have some small optional suggestions

File: doc/modules/ROOT/pages/getting-started.adoc [hunk 4]
```diff
@@ -91,6 +91,28 @@ Or we could change the database we are targeting later:
...
+
+* arrow_disable_server_verification - A flag that indicates that, if the flight client is connecting with
+        TLS, that it skips server verification. If this is enabled, all
```
Comment (nvitucci) [inline doc/modules/ROOT/pages/getting-started.adoc:99]: I would rephrase this a bit. What does "other TLS settings are overridden" mean? Also, when would one want to use this? When a self-signed certificate is used?

File: doc/modules/ROOT/pages/getting-started.adoc (hunk 3 above)
Comment (nvitucci) [reply doc/modules/ROOT/pages/getting-started.adoc:96]: I think it's a good idea. We should also probably refer to it as "Apache Arrow" as we do in the server docs.

File: doc/modules/ROOT/pages/getting-started.adoc (hunk 4 above)
Comment (FlorentinD) [reply doc/modules/ROOT/pages/getting-started.adoc:99]: We are wrapping options that you can configure on the pyArrow flight client (https://arrow.apache.org/docs/python/generated/pyarrow.flight.FlightClient.html#pyarrow.flight.FlightClient).

I will try to find out when it makes sense :)

Comment (FlorentinD) [reply doc/modules/ROOT/pages/getting-started.adoc:99]: Reading more i think its mainly ease of use in the arrow setup
